dcc-storage-client/pom.xml

```diff
@@ -187,7 +188,8 @@ ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF S
     <dcc-common.version>4.0.9</dcc-common.version>
 
     <!-- Versions - Genomics -->
-    <htsjdk.version>1.138</htsjdk.version>
+    <!--  <htsjdk.version>1.138</htsjdk.version> -->
+    <htsjdk.version>2.1.0</htsjdk.version>

```

Comment:
Curious as to why this was needed. Are you happy with the results? If so, let's remove the comment and use git log instead.